Bottom: 980a740beee9d46dbd8d0138d171fc658d75e25c
Top:    60a3be9a19a55eda52190f2b5aba845b983c11be
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 15:44:21 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d9641aa..c21808b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1199,7 +1199,7 @@ ghid_request_debug_draw (void)
 void
 ghid_flush_debug_draw (void)
 {
-//  gdk_window_process_all_updates ();
+  gdk_window_process_all_updates ();
 }
 
 void
